libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -962,6 +1163,28 @@ AP_GPS_UBLOX::_parse_gps(void)
                 if (_hardware_generation != UBLOX_F9) {
                     // need to ensure time mode is correctly setup on F9
                     _unconfigured_messages |= CONFIG_TMODE_MODE;
+
+                    // see if RTK moving baseline base is wanted
+                    if (role == AP_GPS::GPS_ROLE_MB_BASE) {
+                        rtcm3_parser = new RTCM3_Parser;
+                        if (rtcm3_parser == nullptr) {
+                            GCS_SEND_TEXT(MAV_SEVERITY_ERROR,
```

Comment:
This situation should also be reflected within the drivers `is_healthy()` check, so that we can stop arming, and give meaningful errors with this.

Reply:
fixed